### PR TITLE
Refactoring-b2b

### PR DESCRIPTION
**Pull request by himanshu-pal-metacube**
## PR Description

CC-16760 b2b Refactoring

### CODE REVIEW

**Comment by himanshu-pal-metacube** [comment]
Here is Result file of all test
[results.zip](https://github.com/spryker/robotframework-suite-tests/files/8916103/results.zip)
s

File: resources/environments/environments_api_b2b.json [hunk 1]
```diff
@@ -363,37 +489,55 @@
...
-        "discount_id_3_voucher_code": "sprykerca9j",
-        "discount_id_4_name": "10% off Safescan",
-	    "discount_id_4_description": "Get a 10% discount on all Safescan brand products with voucher code",
-        "discount_id_4_voucher_code": "safescanqu1pu4ga1z",
+        
+        "shipment_method":{
+            "carrier_name": "DHL",
+            "name": "Standard",
+            "delivery_date": "2030-12-31",
+            "name_2": "Express"
```
Comment (OKrivtsova) [inline resources/environments/environments_api_b2b.json:497]: this section should be called 'shipment'
inside we should have shipment_method1 - Standard, shipment_method2 - Exprtess

File: resources/environments/environments_api_b2b.json [hunk 2]
```diff
@@ -6,294 +6,415 @@
...
-        "yves_user_1st_shopping_list_id": "cf032865-d1ad-5e27-803a-423bd15ced66",
-        "yves_user_1st_shopping_list_name": "Newcomers",
-        "yves_user_2nd_shopping_list_name": "Weekly office",
-        "yves_user_3rd_shopping_list_name": "Monthly office"
+        "shopping_list_1st":{
+            "id": "cf032865-d1ad-5e27-803a-423bd15ced66",
+            "name": "Newcomers"
+        },
+        "shopping_list_name_2": "Weekly office",
+        "shopping_list_name_3": "Monthly office"
```
[the remark this answers is not in the document]
Comment (himanshu-pal-metacube) [reply resources/environments/environments_api_b2b.json]: Changed

File: resources/environments/environments_api_b2b.json (hunk 1 above)
Comment (himanshu-pal-metacube) [reply resources/environments/environments_api_b2b.json:501]: Changed

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**